src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -7727,7 +7745,7 @@ public void ListPattern_SetOnlyIndexers()
 
 class C
 {
-    public int Length { set { } }
+    public int Length => 0;
```

Comment:
>=> 0; [](http://example.com/codeflow?start=22&length=5)

It is not clear to me why the scenario got changed. I think we should still test it.